workshops/py_Christmas/README.md

```diff
@@ -0,0 +1,381 @@
+
+Last function in this file to make the tiny balls on either sides of the boxes (green part) pf the christmas tree.
+Go through the code. You probably will understand it pretty easily.
+```python
+def balls(turtle, color, x, y, radius):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x, y)
+    turtle.pendown()
+    turtle.begin_fill()
+    turtle.circle(radius)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+
+``` 
+Yes, ******turtle****** library has a circle function. Made our job pretty easy, didn't it ? 
+
+So yeah, that's it ... our funtions to make the figures is ready. Wanna see the whole code again ? Take a look. 
+
+```python
+import turtle
+def box1 (turtle, color, x, y, width, height):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x,y)
+    turtle.pendown()
+    turtle.begin_fill()
+    for i in range(2):
+        turtle.forward(width)
+        turtle.left(90)
+        turtle.forward(height)
+        turtle.left(90)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+
+
+
+def star(turtle, color, x, y, size):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x, y)
+    turtle.pendown()
+    turtle.begin_fill()
+    for i in range (5):
+        turtle.forward(size)
+        turtle.right(144)
+        turtle.forward(size)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+
+
+def balls(turtle, color, x, y, radius):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x, y)
+    turtle.pendown()
+    turtle.begin_fill()
+    turtle.circle(radius)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+``` 
+All good ? 
+
+# Now, lets move on to Next file : ChristmasTree.py
+
+
+Now open the main.py 
+
+In this page we are gonna make a tree stem made of rectangle then we are gonna make multiple rectangles stacked above each other to make it look like a christmas tree. 
+Shall we begin ?
+
+```python
+from turtle import *
+from Figures import *
+from random import randint
+
+```
+Here, we import all fuctions from 'turtle', all functions from 'Figures' and finally 'randint' function from 'random' so that we can use it to generate a random integer.
+Why do we need a random integer ? We'll see .
+
+
+```python
+speed(0)
+
+bg = turtle.Screen()
+bg.bgcolor("#99004D") #Setting back ground color
+
+y = -100
+width = 240
+
+```
+******speed()****** function determines the speed of the turtle, ie, the speed in which the figures get drawn on the screen. When we set that to zero it moves as fast as it can. 
+Next we set the background color to maroon using th hexcode.
+Now decide a value for ******y******, which is basically trial and error. Set some random value first and then iterate to find the right vaue which brings your turtle to the center.
+In Case of ******width******, provide any value you desire. 
```

Comment:
```suggestion
******speed()****** function determines the speed of the turtle, ie, the speed at which the figures get drawn on the screen. When we set that to zero it moves as fast as it can. 
Next, we set the background color to maroon using the hex code.
Now decide a value for ******y******, which is basically trial and error. Set some random value first and then iterate to find the right value which brings your turtle to the center.
In Case of ******width******, provide any value you desire. 
```